HerPortal.UnitTests/Website/Models/HomepageViewModelTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FluentAssertions;
+using HerPortal.BusinessLogic.Models;
+using HerPortal.Models;
+using NUnit.Framework;
+
+namespace Tests.Website.Models;
+
+[TestFixture]
+public class HomepageViewModelTests
+{
+    [Test]
+    public void ShouldShowBannerIfUserHasNotLoggedIn()
+    {
+        // Arrange
+        var user = new User
+        {
+            HasLoggedIn = false,
+        };
+        
+        // Act
+        var viewModel = new HomepageViewModel(user);
+        
+        // Assert
+        viewModel.ShouldShowBanner.Should().BeTrue();
+    }
+    
+    [Test]
+    public void ShouldNotShowBannerIfUserHasLoggedIn()
```

Comment:
Two tests is fine, but be aware that you could also write this as one test with two [TestCase(true, false)] annotations